BattleOfDazaralor/Locales/esES.lua

```diff
@@ -1,4 +1,4 @@
-local L = BigWigs:NewBossLocale("Battle of Dazar'alor Trash", "esES") or BigWigs:NewBossLocale("Battle of Dazar'alor Trash", "esMX")
+local L = BigWigs:NewBossLocale("Basura de Batalla de Dazar'alor", "esES") or BigWigs:NewBossLocale("Basura de Batalla de Dazar'alor", "esMX")
```

Comment:
You're not supposed to change these names. They are the module names.

Reply:
Oops! Sorry I will fix that tomorrow!